climatetree/frontendfinal
# removed ReactTinyLink and better responsiveness (landing page)

### Change type:
- Bug fix

### Description:
- Removed ReactTinyLink from the landing page, replaced with StoryPreview component
- Better responsiveness (landing page)
- Integrate dropdown improvement to map search on the landing page (Evan did for map place search)
- Better import strategy for dynamically importing CSS styles (StoryPreview)

### Trello Card Link:
https://trello.com/c/5byPpIy5/64-refactor-landing-page

### Steps to Verify This PR
- Search place on the landing page under the map tab
- Check the explore section (mobile and desktop)
- @theodoreandrew I removed font-size part inside StoryPreview.profile.css because I think it is too big. Feel free to create a new stylesheet for the stories page.
